docs/markdown/authoritative/backend-remote.md

```diff
@@ -1045,6 +1045,38 @@ Content-Type: text/javascript; charset=utf-8
 {"result":"PONG"}
 ```
 
+### `searchRecords`
+Can be used to search records from the backend. This is used by web api.
+
+* Mandatory: no
+* Parameters: pattern, maxResults
+* Reply: same as (lookup)(backend-remote.md#lookup) or false to indicate failed search
```

Comment:
wrong markdown syntax (should be `[link text](url)`). Also, it's on the same page so the following should work: `[lookup](#lookup)`